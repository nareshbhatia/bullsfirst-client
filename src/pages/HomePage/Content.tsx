import React from 'react';

export const Content = () => {
  return (
    <div data-testid="home-content" className="container">
      <main className="max-w-mobile pt-4 pb-2">
        <p className="m-0">Every account comes with:</p>
        <ul className="m-0">
          <li>0-second execution</li>
          <li>Portfolio insurance</li>
          <li>1 popsicle</li>
        </ul>
        <p className="mt-2">We love to trade, and we hope you do too!</p>
        <p>
          Now you can express that love with the latest trading platform from
          Bullsfirst. We guarantee a 0-second trade execution, and will give you
          a free popsicle if your trade doesn’t go through in this time period.
        </p>
        <p>
          <i>
            On a side-note, we just bought a big freezer and 10,000 popsicles.
            Thankfully, we’ve hedged all popsicle expenses for the next two
            years! (Commodity experts say popsicles are on the rise).
          </i>
        </p>
      </main>
    </div>
  );
};
