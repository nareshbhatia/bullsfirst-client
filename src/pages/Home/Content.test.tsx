import React from 'react';
import { render } from '../../test/test-utils';
import { Content } from './Content';

describe('<Content />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Content />);
    expect(asFragment()).toMatchSnapshot();
  });
});
