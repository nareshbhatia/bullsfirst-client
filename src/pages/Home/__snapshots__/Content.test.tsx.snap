// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="home-content"
  >
    <main
      class="max-w-mobile pt-4 pb-2"
    >
      <p
        class="m-0"
      >
        Every account comes with:
      </p>
      <ul
        class="m-0"
      >
        <li>
          0-second execution
        </li>
        <li>
          Portfolio insurance
        </li>
        <li>
          1 popsicle
        </li>
      </ul>
      <p
        class="mt-2"
      >
        We love to trade, and we hope you do too!
      </p>
      <p>
        Now you can express that love with the latest trading platform from Bullsfirst. We guarantee a 0-second trade execution, and will give you a free popsicle if your trade doesn’t go through in this time period.
      </p>
      <p>
        <i>
          On a side-note, we just bought a big freezer and 10,000 popsicles. Thankfully, we’ve hedged all popsicle expenses for the next two years! (Commodity experts say popsicles are on the rise).
        </i>
      </p>
    </main>
  </div>
</DocumentFragment>
`;
